### PR TITLE
Class 3 HW - ex_teddit_basic.rb

### PR DESCRIPTION
**Pull request by juliekwak**

### CODE REVIEW

File: JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  puts "Upvote: " + calculate_votes(category).to_s
+  upvote = calculate_votes(category)
+
+  story_hash = {}
+  story_hash[:title] = title
+  story_hash[:category] = category
+  story_hash[:upvote] = upvote
+  story_hash
+
+end
```
Comment (dladowitz) [inline JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb]: This method is called `get_story` so I'd limit it do doing one thing, getting the story and creating a hash. 
I'd make another method called `print_story` that does the printing.

File: JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#3 -  Save every story to a collection (each story should be a hash)
+#4 -  Print each stories title, category and upvotes to the screen
+
+
+$stories = {}
+
+def get_story(title, category)
+  puts "The title is #{title} and the category is #{category}"
+  puts "Upvote: " + calculate_votes(category).to_s
+  upvote = calculate_votes(category)
```
Comment (dladowitz) [inline JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb]: Great job calling the `calculate_votes` method inside the `get_story` method. 
You call it twice in a row though. better would be to call it once, save the result to `upvote` and then use `upvote` anywhere you want the upvote count.

**Comment by dladowitz** [comment]
Nice work on teddit_basic. :+1:

File: JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb (hunk 2 above)
Comment (juliekwak) [inline JulieKwak-juliekwak/03_Collections_Loops_HW/ex_teddit_basic.rb]: Makes sense. Thanks!

**Comment by juliekwak** [comment]
thank you!

**Pull request merged**